guides/common/modules/proc_enabling-insights-on-your-project-server.adoc

```diff
@@ -0,0 +1,24 @@
+[id='enabling-insights-on-your-{project-context}-server']
+= Enabling Insights on your {ProjectServer}
+
+To maintain your {ProjectServer}, and improve your ability to monitor and diagnose problems you might have with {Project}, install Red{nbsp}Hat Insights on {ProjectServer} and register {ProjectServer} with Red{nbsp}Hat Insights.
+
+.Procedure
+
+. Install Red{nbsp}Hat Insights on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {project-package-install} insights-client
+----
++
+. Register {ProjectServer} with Red{nbsp}Hat Insights:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --register-with-insights
+----
+
+.Next steps
+* Optional: Change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
```

Comment:
What does this do? I was not aware that we recommended using insights-client on the Satellite base system. I was also not aware of a `--register-with-insights` option in the installer. It seems like this procedure would only register the Satellite base system to Insights, and doesn't have anything to do with actually using Insights within Satellite.

To use hosted Insights, what should we recommend @ehelms ? I'd think it's just making sure you have host_registration_insights global param set (which is on by default), but I don't know about the installer option so I'll defer to you.

Reply:
It looks like this is very old content, added when the guide was first upstreamized with https://github.com/theforeman/foreman-documentation/commit/8e2851815e7  I'd be happy to change anything that needs updating.